src/app/activity/activity.service.ts

```diff
@@ -111,7 +114,11 @@ export class ActivityService {
             type: 'Assessment',
             contextId: contextIds[element[element.model].id] || 0,
             loadingStatus: true,
-            isForTeam: element[element.model].is_team
+            isForTeam: element[element.model].is_team,
+            // @TODO need to update after api ready
+            dueDate: this.utils.validateDueDates(element[element.model].deadline),
```

Comment:
Here, as an attribute of `Task`, `dueDate` should be just a date string, instead of how we display it.
We could define how we display it in html file